corehq/apps/users/models.py

```diff
@@ -224,6 +224,12 @@ class HqPermissions(DocumentSchema):
 
     manage_domain_alerts = BooleanProperty(default=False)
 
+    view_commcare_analytics = BooleanProperty(default=False)
+    edit_commcare_analytics = BooleanProperty(default=False)
+
+    commcare_analytics_roles = BooleanProperty(default=False)
+    commcare_analytics_roles_list = StringListProperty(default=[])
```

Comment:
note to self:
Add migration for these once staging is unblocked on conflicts with multiple migrations in parallel to users app.

Reply:
In the `Technical Summary` it says 
> The migration added simply creates the SQL equivalent for the couch model attributes where new permissions are added.

I presume this is migration you're referring to above?

Reply:
Hey @Charl1996 
Yes. I had the migration added but then removed it in effort to resolve the staging conflict but there were more conflicts, so I will probably just add it back anyway. Just waiting on some more information on the other PR the conflict was with.
The way the migration is, the code can still be deployed without it and the staging environment won't break.